pkg/chain/local/local.go

```diff
@@ -26,6 +26,8 @@ import (
 type Chain interface {
 	chain.Handle
 
+	OperatorAddress() common.Address
+
```

Comment:
Should we refactor functions below to accept `chain.ID` instead of `common.Address?

Reply:
(Appearing in my capacity as open source contributor rather than employee..... it didn't seem worth the effort since the functions are just for local testing, rather than for external interface exposure.)